### PR TITLE
Add Rails 5.1 support

### PR DESCRIPTION
**Pull request by tvdeyen**
Builds on top of #18

### CODE REVIEW

**Comment by tvdeyen** [comment]
@dhiraj14 needs to be rebased now that we made a change to how we store the `searchable` attribtue

**Comment by dhiraj14** [comment]
> @dhiraj14 needs to be rebased now that we made a change to how we store the `searchable` attribtue

Sounds good. Thanks @tvdeyen

File: spec/dummy/config/boot.rb [hunk 1]
```diff
@@ -1,3 +1,3 @@
-ENV["BUNDLE_GEMFILE"] ||= File.expand_path("../../../Gemfile", __dir__)
+ENV['BUNDLE_GEMFILE'] ||= File.expand_path('../Gemfile', __dir__)
```
Comment (tvdeyen) [inline spec/dummy/config/boot.rb]: This needs to be reverted because the `Gemfile` is in the root of the gem and not in the root of the `spec/dummy` app.

**Review by tvdeyen — COMMENTED** [review]
There is one change that needs to be reverted. Otherwise we cannot use the dummy app for local dev

File: spec/dummy/config/boot.rb (hunk 1 above)
Comment (dhiraj14) [reply spec/dummy/config/boot.rb]: @tvdeyen Updated

**Pull request merged**